### ISSUE TITLE
"procalc.app" would like to control this computer using accessibility features

### ISSUE THREAD

**Issue by RichieB2B** [RobJinman/pro_office_calc#3]
Is this app safe to use on OS X?
<img width="529" alt="screenshot 2018-12-08 10 42 47" src="https://user-images.githubusercontent.com/1461970/49684619-6a68d280-fad6-11e8-9195-2a3e70456e39.png">

**Comment by RobJinman** [comment]
Yes, it's perfectly safe to use. I'm not sure what that dialog means (not being a regular Mac user myself). I'll investigate. Thanks

**Comment by RichieB2B** [comment]
I received the message when messing with (clicking/selecting text/dragging) the About menu on my first launch. I selected _Deny_ in the pop-up and the app is working fine.
